lib/index.js

```diff
@@ -76,11 +76,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         }
 
         const {actor} = req.user;
-        await brAuthnToken.set({
+        const result = await brAuthnToken.set({
```

Comment:
I guess we're going to need `serviceId` here.

Reply:
+1